lib/Server.js

```diff
@@ -382,10 +382,10 @@ class Server {
       static: () => {
         this.setupStaticFeature();
       },
-      serveIndex: () => {
+      staticServeIndex: () => {
         this.setupServeIndexFeature();
```

Comment:
And let's rename method names: `this.setupStaticServeIndexFeature`, as I written above :1st_place_medal:

Reply:
> And let's rename method names: `this.setupStaticServeIndexFeature`, as I written above

Just did that :+1: